src/components/layout/Footer.js

```diff
@@ -40,12 +45,15 @@ export const ContentContainer = styled.div`
 
 export const LeftContent = styled.div`
   height: 100%;
+  display: flex;
+  flex-direction: column;
   text-align: center;
+  align-items: center;
   max-width: 200px;
 
   @media (min-width: 1024px) {
     text-align: left;
-    flex-direction: row;
+    align-items: flex-start;
     min-width: 200px;
```

Comment:
Do these change anything (this plus the above)?

Reply:
removed from code